Source/WebKit/UIProcess/Cocoa/_WKWarningView.mm

```diff
@@ -260,6 +260,17 @@ static CGSize buttonSize(ButtonType *button)
 #endif
 }
 
+static WarningItem backgroundColorForWarningType(const WebKit::BrowsingWarning& warning)
```

Comment:
This doesn't make any sense.  I would expect this to return a color, but instead it returns WarningItem::SafeBrowsingBackground even if that isn't accurate.